IBAnimatable/TransitionAnimationType.swift

```diff
@@ -202,6 +228,24 @@ extension TransitionAnimationType {
     case none
     case open
     case close
+
+    var opposite: HollowState {
+      switch self {
+      case .close: return .open
+      case .open: return .close
+      case .none:
+        fatalError("No opposing HollowState for HollowState.none")
+      }
+    }
+
+    var opposingGesture: InteractiveGestureType.GestureDirection {
+      switch self {
+      case .open: return .close
+      case .close: return .open
+      case .none:
+        fatalError("No opposing GestureDirection for HollowState.none")
```

Comment:
I thought about this too but returning `.none` seemed counterintuitive as it's not an opposite.

I thought `fatalError` would help catch some programmer errors but I can change this to the requested functionality.